Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/DeletionProcessGracePeriod/HandlerTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Backbone.BuildingBlocks.Application.PushNotifications;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Application.Identities.Commands.DeletionProcessGracePeriod;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.Infrastructure.PushNotifications.DeletionProcess;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.DeletionProcessGracePeriod;
+public class HandlerTests
+{
+    [Fact]
+    public async Task No_Active_Deletion_Processes()
```

Comment:
This name doesn't match the implementation.
```suggestion
    public async Task No_identities_with_an_approved_deletion_process_exist()
```